Slider/Assets/Scenes/Caves.unity

```diff
@@ -461391,36 +461787,37 @@ PrefabInstance:
       objectReference: {fileID: 0}
     - target: {fileID: 3652435011426292518, guid: eb8cd11432469439eb214403b87b5b92, type: 3}
       propertyPath: conds.Array.data[6].dialogueChain.Array.data[2].dialogue
-      value: ' rearranging...'
+      value: a powerful tool that can break apart and move the world...
```

Comment:
i want these lines to be short and punchy bc she says them during the cave cutscene when everything is going woop woop woop